### PR TITLE
[pt] Translate shortcode snippets /languages/exporters

### LINKED ISSUE #5592: [pt] localization for shortcode snippets on /languages/exporters
**What needs to be changed?** 

This issue tracks the Portuguese localization of the shortcode snippets under [layouts/shortcodes/docs/languages/exporters/](https://github.com/open-telemetry/opentelemetry.io/tree/main/layouts/shortcodes/docs/languages/exporters):

- layouts/shortcodes/docs/languages/exporters/intro.md
- layouts/shortcodes/docs/languages/exporters/jaeger.md
- layouts/shortcodes/docs/languages/exporters/outro.md
- layouts/shortcodes/docs/languages/exporters/prometheus-setup.md
- layouts/shortcodes/docs/languages/exporters/zipkin-setup.md

## Pull Request

*  https://github.com/open-telemetry/opentelemetry.io/pull/5593

### PR DESCRIPTION
**Pull request by vitorvasc**
* Add new translations:
   - layouts/shortcodes/pt/docs/languages/exporters/intro.md
   - layouts/shortcodes/pt/docs/languages/exporters/jaeger.md
   - layouts/shortcodes/pt/docs/languages/exporters/outro.md
   - layouts/shortcodes/pt/docs/languages/exporters/prometheus-setup.md
   - layouts/shortcodes/pt/docs/languages/exporters/zipkin-setup.md
* Fixes translations for snippets on:
   - content/pt/docs/concepts/components.md (https://opentelemetry.io/pt/docs/concepts/components/#exporters)
   - content/pt/docs/languages/go/exporters.md (https://opentelemetry.io/pt/docs/languages/go/exporters/)

Fixes #5592

### CODE REVIEW

**Review by EzzioMoreira — APPROVED** [review]
LGTM!!!
Parabéns @vitorvasc 🤘

File: layouts/shortcodes/pt/docs/languages/exporters/intro.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+{{/* cSpell:ignore cond */ -}} Envie dados de telemetria para o [OpenTelemetry Collector](/docs/collector/) para garantir que estes dados serão exportados corretamente. A utilização de um Collector em ambientes produtivos é a melhor prática. Para visualizar os dados de telemetria que foram gerados, realize a exportação para um ou mais _backends_ como [Jaeger](https://jaegertracing.io/), [Zipkin](https://zipkin.io/),
```
Comment (maryliag) [inline layouts/shortcodes/pt/docs/languages/exporters/intro.md:1]:
```suggestion
{{/* cSpell:ignore cond */ -}} Envie dados de telemetria para o [OpenTelemetry Collector](/docs/collector/) para garantir que estes dados sejam exportados corretamente. A utilização de um Collector em ambientes de produção é a melhor prática. Para visualizar os dados de telemetria que foram gerados, exporte-os para um _backend_ como [Jaeger](https://jaegertracing.io/), [Zipkin](https://zipkin.io/),
```

File: layouts/shortcodes/pt/docs/languages/exporters/intro.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+{{ $name = (index $.Site.Data.instrumentation $lang).name -}}
+
+## Exportadores disponíveis {#available-exporters}
+
+O registro oferece uma [lista de exportadores para {{ $name }}][reg].
+
+{{ else -}}
+
+O registro oferece uma [lista de exportadores específicos para a linguagem][reg].
```
Comment (maryliag) [inline layouts/shortcodes/pt/docs/languages/exporters/intro.md:19]:
```suggestion
O registro oferece uma [lista de exportadores específicos de linguagem][reg].
```

File: layouts/shortcodes/pt/docs/languages/exporters/intro.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+<div class="alert alert-info" role="alert"><h4 class="alert-heading">Nota</h4>
+
+Caso você esteja utilizando [instrumentação sem código](/docs/zero-code/{{ $l }}), você poderá aprender a configurar os exporters através do [Guia de Configurações](/docs/zero-code/{{ $l }}/configuration/).
+
+</div>
+
+{{ end -}}
+
+{{/*
+ below list needs to grow until all languages are updated to a consistent structure.
```
Comment (maryliag) [inline layouts/shortcodes/pt/docs/languages/exporters/intro.md:51]:
```suggestion
 a lista a seguir precisa crescer até que todas as línguas sejam atualizadas para uma estrutura consistente.
```

File: layouts/shortcodes/pt/docs/languages/exporters/intro.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
+{{/* cSpell:ignore cond */ -}} Envie dados de telemetria para o [OpenTelemetry Collector](/docs/collector/) para garantir que estes dados serão exportados corretamente. A utilização de um Collector em ambientes produtivos é a melhor prática. Para visualizar os dados de telemetria que foram gerados, realize a exportação para um ou mais _backends_ como [Jaeger](https://jaegertracing.io/), [Zipkin](https://zipkin.io/),
+[Prometheus](https://prometheus.io/), ou um backend
```
Comment (maryliag) [inline layouts/shortcodes/pt/docs/languages/exporters/intro.md:2]:
```suggestion
[Prometheus](https://prometheus.io/), ou um _backend_ 
```

File: layouts/shortcodes/pt/docs/languages/exporters/prometheus-setup.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+Para enviar dados de métricas para o [Prometheus](https://prometheus.io/), é possível
+[ativar o OTLP Receiver do Prometheus](https://prometheus.io/docs/prometheus/latest/feature_flags/#otlp-receiver)
+e utilizar o [exportador OTLP](#otlp) ou utilizar o exportador do Prometheus, um `MetricReader` que inicia um servidor HTTP e coleta métricas, serializando para o formato de texto do Prometheus sob demanda.
+
+### Configuração do Backend {#prometheus-setup}
+
+<div class="alert alert-info" role="alert"><h4 class="alert-heading">Nota</h4>
+
+Caso já possua o Prometheus ou um backend compatível com Prometheus configurado, poderá pular esta seção e configurar as dependências do exportador [Prometheus](#prometheus-dependencies) ou [OTLP](#otlp-dependencies) para a sua aplicação.
```
Comment (maryliag) [inline layouts/shortcodes/pt/docs/languages/exporters/prometheus-setup.md:11]:
```suggestion
Caso já possua o Prometheus ou um _backend_ compatível com Prometheus configurado, poderá pular esta seção e configurar as dependências do exportador [Prometheus](#prometheus-dependencies) ou [OTLP](#otlp-dependencies) para a sua aplicação.
```

File: layouts/shortcodes/pt/docs/languages/exporters/zipkin-setup.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+## Zipkin
+
+### Configuração do Backend {#zipkin-setup}
+
+<div class="alert alert-info" role="alert"><h4 class="alert-heading">Nota</h4>
+
+Caso já possua o Zipkin ou um backend compatível com Zipkin configurado, poderá pular esta seção e configurar as [dependências do exportador Zipkin](#zipkin-dependencies) para a sua aplicação.
```
Comment (maryliag) [inline layouts/shortcodes/pt/docs/languages/exporters/zipkin-setup.md:7]:
```suggestion
Caso já possua o Zipkin ou um _backend_ compatível com Zipkin configurado, poderá pular esta seção e configurar as [dependências do exportador Zipkin](#zipkin-dependencies) para a sua aplicação.
```

File: layouts/shortcodes/pt/docs/languages/exporters/intro.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+      exporters: [debug]
+```
+
+Em seguida, execute o Collector em um contêiner Docker através do seguinte comando:
+
+```shell
+docker run -p 4317:4317 -p 4318:4318 --rm -v $(pwd)/collector-config.yaml:/etc/otelcol/config.yaml otel/opentelemetry-collector
+```
+
+Em alguns instantes, o Collector estará disponível para receber dados de telemetria via OTLP. Mais tarde, você também poderá querer [configurar o Collector](/docs/collector/configuration) para enviar os seus dados de telemetria para o seu _backend_ de observabilidade.
```
Comment (maryliag) [inline layouts/shortcodes/pt/docs/languages/exporters/intro.md:100]:
```suggestion
Este Collector agora é capaz receber dados de telemetria via OTLP. Mais tarde, você também poderá [configurar o Collector](/docs/collector/configuration) para enviar os seus dados de telemetria para o seu _backend_ de observabilidade.
```

File: layouts/shortcodes/pt/docs/languages/exporters/outro.md [hunk 8]
```diff
@@ -0,0 +1,11 @@
+{{ $lang := .Get 0 -}} {{ $spanExporterInterfaceUrl := .Get 1 }}
+
+## Exportadores personalizados {#custom-exporters} 
+
+Por fim, também é possível escrever o seu próprio exportador. Para mais informações, consulte [SpanExporter Interface na documentação da API]({{ $spanExporterInterfaceUrl }}).
+
+## Agrupamento de Trechos e Registros de Log {#batching-span-and-log-records}
+
+O SDK do OpenTelemetry fornece um conjunto de processadores de Trechos e Registros de Log padrão, que permitem emitir Trechos um-a-um ("simples") ou em lotes. O uso de agrupamentos é recomendado, mas caso não deseje agrupar seus Trechos ou Registros de Log, é possível utilizar um processador simples da seguinte forma:
```
Comment (maryliag) [inline layouts/shortcodes/pt/docs/languages/exporters/outro.md:9]:
```suggestion
O SDK do OpenTelemetry fornece um conjunto de processadores padrão de trechos e registros de log, que permitem emitir trechos um-a-um ("simples") ou em lotes. O uso de agrupamentos é recomendado, mas caso não deseje agrupar seus trechos ou registros de log, é possível utilizar um processador simples da seguinte forma:
```

File: layouts/shortcodes/pt/docs/languages/exporters/outro.md [hunk 9]
```diff
@@ -0,0 +1,11 @@
+{{ $lang := .Get 0 -}} {{ $spanExporterInterfaceUrl := .Get 1 }}
+
+## Exportadores personalizados {#custom-exporters} 
+
+Por fim, também é possível escrever o seu próprio exportador. Para mais informações, consulte [SpanExporter Interface na documentação da API]({{ $spanExporterInterfaceUrl }}).
+
+## Agrupamento de Trechos e Registros de Log {#batching-span-and-log-records}
```
Comment (maryliag) [inline layouts/shortcodes/pt/docs/languages/exporters/outro.md:7]:
```suggestion
## Agrupamento de trechos e registros de log {#batching-span-and-log-records}
```

File: layouts/shortcodes/pt/docs/languages/exporters/prometheus-setup.md [hunk 10]
```diff
@@ -0,0 +1,39 @@
+## Prometheus
+
+Para enviar dados de métricas para o [Prometheus](https://prometheus.io/), é possível
```
Comment (maryliag) [inline layouts/shortcodes/pt/docs/languages/exporters/prometheus-setup.md:3]:
```suggestion
Para enviar dados de métricas para o [Prometheus](https://prometheus.io/), você pode
```

File: layouts/shortcodes/pt/docs/languages/exporters/prometheus-setup.md [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+[ativar o OTLP Receiver do Prometheus](https://prometheus.io/docs/prometheus/latest/feature_flags/#otlp-receiver)
+e utilizar o [exportador OTLP](#otlp) ou utilizar o exportador do Prometheus, um `MetricReader` que inicia um servidor HTTP e coleta métricas, serializando para o formato de texto do Prometheus sob demanda.
```
Comment (maryliag) [inline layouts/shortcodes/pt/docs/languages/exporters/prometheus-setup.md:5]:
```suggestion
e utilizar o [exportador OTLP](#otlp) ou você pode utilizar o exportador do Prometheus, um `MetricReader` que inicia um servidor HTTP e coleta métricas, serializando para o formato de texto do Prometheus sob demanda.
```

**Review by maryliag — COMMENTED** [review]
Parabéns pelo trabalho, adicionei algumas recomendações 😄

**Review by chalin — CHANGES_REQUESTED** [review]
Thanks for this. Please:

- Wait for #5676 to be approved and merge
- Then rebase this PR from `main` and fix any formatting issues

**Comment by vitorvasc** [comment]
Thank you @chalin! Will keep an eye on it.

**Comment by vitorvasc** [comment]
> Thanks for this. Please:
> 
> * Wait for [[CI] Prettify es and pt shortcodes too #5676](https://github.com/open-telemetry/opentelemetry.io/pull/5676) to be approved and merge
> * Then rebase this PR from `main` and fix any formatting issues

@chalin Done!

**Comment by svrnm** [comment]
@chalin PTAL!

**Review by maryliag — APPROVED** [review]

**Review by chalin — APPROVED** [review]
LGTM, TY ✨

**Pull request merged**